package com.rm.dto;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class TopProductResponse {

    private String productName;

    private Long quantitySold;
}
